spacelift/resource_stack.go

```diff
@@ -352,6 +352,12 @@ func resourceStack() *schema.Resource {
 				Optional:    true,
 				Default:     false,
 			},
+			"enable_sensitive_outputs_upload": {
+				Type:        schema.TypeBool,
+				Description: "Indicates whether sensitive outputs created by this stack can be uploaded to Spacelift to be used by Stack Dependency references. Triggered only when corresponding option is enabled on the Worker Pool used by the Stack as well. Defaults to `true`.",
+				Optional:    true,
+				Default:     true,
```

Comment:
this'll trigger a resource change, right? should this be rather a part of major bump or are we ok with it?

Reply:
No, the default value is true for all the stacks.